src/main/java/org/example/homework5/EvenDigitsSum.java

```diff
@@ -0,0 +1,24 @@
+package org.example.homework5;
+
+public class EvenDigitsSum {
+    public static void main(String[] args) {
+        int sum = getEvenDigitSum(542);
+        System.out.println(sum);
+    }
+
+    private static int getEvenDigitSum(int number) {
+        if (number < 0) {
+            return -1;
+        }
+
+        int sum = 0;
+        while (number != 0) {
+            int digit = number % 10;
+            if (digit % 2 == 0)
+                sum += digit;
+                number /= 10;
```

Comment:
is `number /= 10;` inside the if block? There is an issue with code formatting
I recommend use if with curly brackets {} even if it is one line, it improves readability
In your case this line is not under if condition, which is correct, but when you read it is not obvious.